SUTrivBot/Models/Question.cs

```diff
@@ -91,7 +100,7 @@ public AnswerResponse VerifyAnswer(string answer)
                     return response;
                 }
 
-                if (AnswerAll ?? false)
+                if (AnswersRequired == null)

```

Comment:
When dealing with a non-bool nullable instead of checking if it is null should check if it "HasValue" 
So instead of `if (AnswersRequired == null)` use `if (!AnswersRequired.HasValue)`

Reply:
Reason for that is you can sometimes get null ref exceptions or false negatives using the `==` operator with nullable types.